docs/ARCHITECTURE.MD

```diff
@@ -6,13 +6,13 @@ The actual architecture has two [controllers](https://godoc.org/github.com/kuber
 
 One of the most important responsibilities of the controller is the [Reconcile Loop](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#reconcile-loop). Inside this "loop" the controller will ensure that it has every resource (Kubernetes and OpenShift objects) created and updated accordingly.
 
-We aim to avoid having a huge monolith inside the reconcile loop that does it all. With that in mind, we separated the responsibility of making the Kubernetes and OpenShift API calls to a package that we call [`client`](../pkg/client). Kubernetes/OpenShift resources that the controller need is defined and created inside the [`builder`](../pkg/controller/kogitoapp/builder) package. `Builder` communicates with the `client` package to bind or create the objects in the cluster. The `Controller` also make calls to `client` to perform certain taks during the `reconcile` loop.
+We aim to avoid having a huge monolith inside the reconcile loop that does it all. With that in mind, we separated the responsibility of making the Kubernetes and OpenShift API calls to a package that we call [`client`](../pkg/client). Kubernetes/OpenShift resources, which the controller needs, are defined and created inside the [`resource`](../pkg/controller/kogitoapp/resource) package. `resource` communicates with the `client` package to bind or create the objects in the cluster. The `controller` also make calls to `client` to perform certain tasks during the `reconcile` loop.
```

Comment:
that does what? from user perspective it is not clear.

Reply:
Well, since I wrote this I believe I can answer: this is a small section to the contributors doc. This is intended to help others to understand the operator architecture before submitting PRs. It's not meant for final users, that's why it's in a separate doc.